### PR TITLE
speedup build, clean, bootstrap

### PR DESCRIPTION
**Pull request by mgrybyk**
## Proposed changes

changed commands:
- speed up build a bit by running it in parallel
- speed up clean up a bit by reducing amount of `rimraf` calls

added commands:
- `bootstrap-fast` which is super fast because it is not cleaning up node_modules of each package;
- `build-fast` which is a bit faster because calls `rimraf` only once, but is less safe because there might be some build steps missed in future
- `setup-fast` it is a link to bootstrap-fast and build-fast

### Testing

_Assuming `npm run setup` has been performed_

`npm run bootstrap` - 47s
`npm run bootstrap-fast` - 2s

`npm run build` - before: 19s, after: 14s
`npm run build-fast` - 9s

## Types of changes

- [x] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist

- [x] I have read the [CONTRIBUTING](https://github.com/webdriverio/webdriverio/blob/master/CONTRIBUTING.md) doc
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] I have added necessary documentation (if appropriate)

## Further comments

closing #4231 for now

### CODE REVIEW

**Comment by mgrybyk** [comment]
@christian-bromann done

File: package.json [hunk 1]
```diff
@@ -2,20 +2,24 @@
   "private": true,
   "scripts": {
     "create": "node ./scripts/generateSubPackage",
-    "bootstrap": "npm run clean && lerna bootstrap --no-ci && npm run link",
-    "build": "lerna run build",
+    "bootstrap-full": "npm run clean && lerna bootstrap --no-ci && npm run link",
+    "bootstrap": "run-p clean:* && lerna bootstrap --force-local && npm run link",
```
Comment (christian-bromann) [inline package.json]:
```suggestion
    "bootstrap": "run-s clean:* && lerna bootstrap --force-local && npm run link",
```

I don't think it makes sense to clean packages while lerna is building them

Comment (mgrybyk) [reply package.json]: Only clean is running in parallel, starting from && the next command goes

Comment (mgrybyk) [reply package.json]: Only clean is running in parallel, starting from && the next command goes

Comment (christian-bromann) [reply package.json]: ah right

**Pull request merged**